### ISSUE TITLE
Don't compile & install jpegtran if already installed on the system

### ISSUE THREAD

**Issue by shouze** [imagemin/jpegtran-bin#72]
I've seen this (maybe) related issue... but I wonder why this package can't try to use the already installed system binary if present.

For example, I'm running a project on ARM architecture. The binary is not present (BTW I could make a PR to provide a `linux/armv7/jpegtran` binary).

**Comment by anton-kasperovich** [comment]
I was really surprised when i found out about it... I was trying to build NodeJS Docker image based on Alpine Linux, i installed system package https://pkgs.alpinelinux.org/package/v3.3/main/armhf/libjpeg-turbo-utils and even after installation i got error:
```
  ⚠ spawn /usr/lib/node_modules/jpegtran-bin/vendor/jpegtran ENOENT
  ⚠ jpegtran pre-build test failed
```

And now instead of installing:
```
apk add nodejs libjpeg-turbo-utils
```
I need to install:
```
apk add nodejs build-base file nasm autoconf
```

I believe everyone could imagine how heavy build packages is (+180MB) and the sad part that i'm actually do not need to install them, it's just because of missed system package check...

**Alpine 3.3**
```
# /usr/bin/jpegtran -v
libjpeg-turbo version 1.4.2 (build 20151029)
Copyright (C) 1991-2012 Thomas G. Lane, Guido Vollbeding
Copyright (C) 1999-2006 MIYASAKA Masaru
Copyright (C) 2009 Pierre Ossman for Cendio AB
Copyright (C) 2009-2015 D. R. Commander
Copyright (C) 2009-2011 Nokia Corporation and/or its subsidiary(-ies)
Copyright (C) 2013-2014 MIPS Technologies, Inc.
Copyright (C) 2013 Linaro Limited

Emulating The Independent JPEG Group's software, version 8d  15-Jan-2012
```

**Comment by liupishui** [comment]
![qq 20160712154216](https://cloud.githubusercontent.com/assets/6503804/16759135/11b724b8-4848-11e6-9aff-0466aa91beac.png)

**Comment by fieryvova** [comment]
Will it be resolved any time soon?
I'm using Alpine just like @anton-kasperovich 
It's sad that I have to install build packages when I don't need them.
```
~ # npm install jpegtran-bin@3.1.0
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.48","npm":"2.15.11"})
npm WARN engine deep-extend@0.4.1: wanted: {"node":">=0.12.0","iojs":">=1.0.0"} (current: {"node":"0.10.48","npm":"2.15.11"})
|
> jpegtran-bin@3.1.0 postinstall /root/node_modules/jpegtran-bin
> node lib/install.js

  ⚠ spawn ENOENT
  ⚠ jpegtran pre-build test failed
  ℹ compiling from source
  ✖ Error: touch configure.ac aclocal.m4 configure Makefile.am Makefile.in && ./configure --disable-shared --prefix="/root/node_modules/jpegtran-bin/vendor" --bindir="/root/node_modules/jpegtran-bin/vendor" && make install
Command failed: configure: error: in `/root/node_modules/jpegtran-bin/c1ce0e96-9d40-47a2-8b56-6f551c81e79a':
configure: error: no acceptable C compiler found in $PATH
See `config.log' for more details
```

**Comment by followbl** [comment]
@anton-kasperovich you ever find a work around for this? I've been looking at solutions for this all day and haven't had any luck.

**Comment by torvitas** [comment]
@followbl there is a patch, although @shinnn doesn't want to merge it for some reason:
https://github.com/imagemin/jpegtran-bin/pull/74/files

**Comment by shinnn** [comment]
Yes this should be fixed in [`bin-wrapper`](https://github.com/kevva/bin-wrapper).

**Comment by torvitas** [comment]
Hi again.

I just fixed that problem in bin-wrapper and opened a pull request.
https://github.com/kevva/bin-wrapper/pull/51
I also forked the -bin repositories of yours for the meantime to use my fork of bin-wrapper.

Greetings, Sascha

**Comment by matteocng** [comment]
There is a pending PR to fix this: kevva/bin-wrapper/pull/51.
